Custom-Video-Player/script.js

```diff
@@ -0,0 +1,41 @@
+const toggle = document.querySelector(".toggle");
+const video = document.querySelector(".player__video");
+const progressFill = document.querySelector(".progress__fill");
+const progress = document.querySelector(".progress");
+const playerSlider = document.querySelectorAll(".player__slider");
+const skip = document.querySelectorAll("[data-skip]");
+toggle.addEventListener("click", playPause);
+video.addEventListener("click", playPause);
+function playPause(e) {
+  if (video.paused) {
+    toggle.textContent = "❚ ❚";
+    video.play();
+  } else {
+    toggle.textContent = "►";
+    video.pause();
+  }
+}
+video.addEventListener("timeupdate", progressVideo);
+function progressVideo() {
```

Comment:
try to keep functions declaration in one place

Reply:
ok